src/main/java/com/stripe/net/LiveStripeResponseGetter.java

```diff
@@ -45,26 +45,24 @@ public LiveStripeResponseGetter(StripeResponseGetterOptions options, HttpClient
 
   @Override
   @SuppressWarnings({"TypeParameterUnusedInFormals", "unchecked"})
-  public <T extends StripeObjectInterface> T request(
-      BaseAddress baseAddress,
-      ApiResource.RequestMethod method,
-      String path,
-      Map<String, Object> params,
-      Type typeToken,
-      RequestOptions options,
-      ApiMode apiMode)
+  public <T extends StripeObjectInterface> T request(ApiRequest apiRequest, Type typeToken)
       throws StripeException {
-    String fullUrl = fullUrl(baseAddress, options, path);
+    String fullUrl =
+        fullUrl(apiRequest.getBaseAddress(), apiRequest.getOptions(), apiRequest.getPath());
     StripeRequest request =
-        new StripeRequest(method, fullUrl, params, RequestOptions.merge(this.options, options));
+        new StripeRequest(
```

Comment:
`StripeRequest` is an unfortunate name in the context of this change :/

Reply:
:'(

Reply:
I don't think there's any action that can be taken now. Long term, StripeRequest is fairly "internal" and probably possible to rename or otherwise phase out in the future.